README.md

```diff
@@ -1267,90 +1267,132 @@ The `job-id-regex` should contain one capture group while matching against the w
 
 Allows to execute jobs using HTCondor which is a specialized workload management system for compute-intensive jobs created by the Center for High Throughput Computing in the Department of Computer Sciences at the University of Wisconsin-Madison (UW-Madison).
```

Comment:
I agree. I think this can be reduced to just one section that describes config backend and then a subsection with all supported backends so the people know that HtCondor, SGE, etc. configuration is already provided.